### DISCUSSION TITLE
How to eliminate the gaps in the 4 corners of the menu

### DISCUSSION THREAD

**Discussion by ffs-akl** [LBCrion/sfwbar#247] [Q&A] [answered]
Please see the screenshot below, all menus have 4 corners with notches.
![swappy-20241017-143929](https://github.com/user-attachments/assets/4ea64a5c-8ff4-47c8-8f75-0609f68348ca)
[sfwbar.config.txt](https://github.com/user-attachments/files/17407280/sfwbar.config.txt)

**Comment by LBCrion** [comment, marked as answer]
window { background: none };

if you need to specify a background for a bar, use window#barname {
background: ... };

**Reply by ffs-akl** [reply]
I'm sorry,  one more question. what is the name of main bar?
![swappy-20241017-182251](https://github.com/user-attachments/assets/b31d9951-4c8c-49c6-8a69-bf11529bfa65)

**Reply by ffs-akl** [reply]
the name of the main bar is sfwbar.  thank you very much!
